pkg/models/controllers/types.go

```diff
@@ -49,6 +49,7 @@ const (
 	Warning      = "warning"
 	Error        = "error"
 	DisplayName  = "displayName"
+	creator      = "creator"
```

Comment:
Constant names usually start with an uppercase.

Reply:
i don‘t think so ，and start with uppercase in go usually mean it can be visit outside the package， it's not necessary in current scenario